tests/async/test_interception.py

```diff
@@ -328,13 +328,12 @@ async def test_page_route_should_not_work_with_redirects(page, server):
 
 async def test_page_route_should_work_with_redirects_for_subresources(page, server):
     intercepted = []
-    await page.route(
-        "**/*",
-        lambda route: (
-            asyncio.create_task(route.continue_()),
-            intercepted.append(route.request),
-        ),
-    )
+
+    async def handle_route(route) -> None:
+        await route.continue_()
```

Comment:
In Node.js we don't await I think, not sure how it helps with flakiness.

Reply:
You are right, not landing for now.